lib/HttpClient.js

```diff
@@ -343,6 +379,8 @@ function HttpClient(options) {
     this.requestTimeout = options.requestTimeout || false;
     this.headers = options.headers || {};
     this.log = options.log;
+    this.followRedirects = options.followRedirects || false;
+    this.maxRedirects = options.maxRedirects || 5;
```

Comment:
Can you add corresponding "assert.optional" calls at the top of this constructor, please?